static/app/components/events/interfaces/crashContent/exception/sourcemapsWizard.tsx

```diff
@@ -19,7 +19,17 @@ import localStorage from 'sentry/utils/localStorage';
 const LOCAL_STORAGE_KEY = 'issues-sourcemap-wizard-hide-until';
 const DISMISS_TIME = 1000 * 60 * 60 * 24 * 7; // 1 week
 
-const wizardCommand = 'npx @sentry/wizard@latest -i sourcemaps';
+// Use a complete command with org and project slugs
+function getWizardCommand(organization: Organization, projectId: string) {
```

Comment:
shouldn't be better to call it `projectSlug`? it can be not very clear because we do have `projectId` too